src/main/java/medicaldoctor/entities/Patient.java

```diff
@@ -0,0 +1,99 @@
+package medicaldoctor.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import medicaldoctor.core.DatabaseScope;
+import medicaldoctor.util.HashAndSalt;
+import org.hibernate.query.Query;
+
+@Entity
+@Table(name = "Patient", uniqueConstraints = {
+    @UniqueConstraint(columnNames = "Id")
+    , @UniqueConstraint(columnNames = "Full Name")})
+@SuppressWarnings("PersistenceUnitPresent")
+
+public class Patient extends AbstractEntity{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "Id", nullable = false, unique = true)
+    private Integer id;
+
+    @Column(name = "FirstName", nullable = false, length = 100)
+    private String firstName;
+
+    @Column(name = "LastName", nullable = false, length = 100)
+    private String lastName;
+
+    @Column(name = "Age", nullable = false, length = 50, unique = true)
+    private Integer age;
+
+    @Column(name = "Adress", nullable = false, length = 128)
```

Comment:
"Address"

Reply:
also address is split into multiple fields, address, city, state, zip, etc.

Reply:
also there are other fields on the patient entity to be added. see our prototype.